app/models/order_process.rb

```diff
@@ -3,6 +3,9 @@ class OrderProcess < ApplicationRecord
   acts_as_tenant(:tenant)
   acts_as_taggable_on
 
+  belongs_to :before_portfolio_item, :class_name => 'PortfolioItem'
```

Comment:
I'm good with this, just wanted to check with @mkanoor if you are ok with this naming.

Reply:
@gmcculloug Looks good